CHANGELOG.md

```diff
@@ -6,6 +6,30 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 3.4.0    
+
+#### General    
+
+* `FEAT`: add notifications ([#1505](https://github.com/camunda/camunda-modeler/issues/1505))
+* `FEAT`: add extension point for ui plugins ([#1490](https://github.com/camunda/camunda-modeler/issues/1490))
+* `FEAT`: add stable api to access workspace configuration fron plug-ins ([#1490](https://github.com/camunda/camunda-modeler/issues/1425))
+* `FIX`: complete direct editing on save operation ([#1473](https://github.com/camunda/camunda-modeler/issues/1473))
+* `FIX`: modal autofocus ([#1489](https://github.com/camunda/camunda-modeler/pull/1489))
```

Comment:
fix autofocusing for modals?

Reply:
It's this one: https://github.com/camunda/camunda-modeler/pull/1489

Should I remove it?

Reply:
Was just wording recommendation :) We should include it.

Reply:
It's okay to reword the commit message when you prepare a changelog update.